control-plane/pkg/reconciler/broker/namespaced_broker.go

```diff
@@ -297,11 +319,50 @@ func (r *NamespacedReconciler) createManifestFromSystemServiceAccount(broker *ev
 	return unstructuredFromObject(cm)
 }
 
+func (r *NamespacedReconciler) servicesFromSystemNamespace(broker *eventing.Broker) ([]unstructured.Unstructured, error) {
```

Comment:
`reconcileSystemNamespaceServices` ?

Sorry, I know those are nits... but wanted to ask

Reply:
it is not really reconciling, it is returning a list of objects.